### ISSUE TITLE
How to get the same ssim value as the paper?

### ISSUE THREAD

**Issue by GuideWsp** [thstkdgus35/EDSR-PyTorch#251]
I have tried some method to calc the ssim value with your pretrained models, but i can't get the same value as the paper.
1. code from https://github.com/xinntao/BasicSR/blob/master/metrics/calculate_PSNR_SSIM.m;
2. code from https://github.com/fperazzi/proSR/blob/master/lib/prosr/metrics.py;

the value of ssim from 1 is 0.8263, while the paper say 0.9017.

I have tried metric it on rgb and y(y from ycbcr), but the results are all different from the paper.

so, could you provide your ssim code? thanks.

**Comment by Doreenqiuyue** [comment]
Do you get the same ssim value as the paper now?
